nixos/modules/services/monitoring/scrutiny-collector.nix

```
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+in
+{
+  options.services.scrutiny-collector = {
+    enable = mkEnableOption (mdDoc "scrutiny-collector");
```

```suggestion
    enable = mkEnableOption "scrutiny-collector";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557